nengo_gui/namefinder.py

```diff
@@ -22,9 +22,12 @@ def find_names(self, net):
         classes = (nengo.Node, nengo.Ensemble, nengo.Network,
                    nengo.Connection)
 
+        private = False
+        in_lists = False
```

Comment:
Also, clarity > efficiency as long as it is not a part of the code which runs really a lot.